docs/pages/app.md

```diff
@@ -87,3 +93,28 @@ function handleURL(context) {
 Open the following link to display a _Hello World_ message within an open document. Provide any URL encoded string with the `msg` query parameter.
 
 [`sketch://plugin/com.example.sketch.messenger/message.show?msg=Hello%20World`](sketch://plugin/com.example.sketch.messenger/message.show?msg=Hello%20World)
+
+## Add a Remote Library <a name="addremotelibrary"></a>
+
+You can add a Remote Library to Sketch by providing a url to an appcast file.
+
+### Parameters
+
+Provide an `encoded appcast url` parameter to the `add-library` command.
+
+```sh
+sketch://add-library?url=AppCastURL
+```
+
+#### Example
+
+Open the link below to add a Remote Library to Sketch
+
+```sh
+sketch://add-library?url=https%3A%2F%2Fdeveloper.apple.com%2Fdesign%2Fdownloads%2Fsketch.rss
+```
+
+## Related resources
+
+- [Sharing a Library](https://www.sketch.com/docs/libraries/#sharing-libraries)
+- [Appcast documentation](https://sparkle-project.org/documentation/)
```

Comment:
Let's drop this. Both links are very specific to the `add-library` command where both links are included now.